packages/deb/hazelcast/DEBIAN/control

```diff
@@ -1,6 +1,6 @@
 Package: hazelcast
 Source: hazelcast
-Version: 4.2020.10
+Version: 5.2021.8
```

Comment:
I don't think you need to change these versions. They are changed by the GH action jobs when releasing.